UserLeapKit.xcframework/ios-arm64_x86_64-simulator/UserLeapKit.framework/sprig_web_assets.bundle/snippet.js

```diff
@@ -42,10 +47,11 @@ Sprig('addListener', 'sdk.ready', () => {
 Sprig('addListener', 'survey.height', (payload) => {
     window.webkit.messageHandlers.sprigWebController.postMessage({type: 'setHeight', height: payload.contentFrameHeight.toString() });
 });
-Sprig.mobileTrackEvent = async (event, userId, partnerAnonymousId, callbackId) => {
+Sprig.mobileTrackEvent = async (event, userId, partnerAnonymousId, properties, callbackId) => {
     const payload = { eventName: event };
     if (userId) payload.userId = userId;
     if (partnerAnonymousId) payload.anonymousId = partnerAnonymousId;
+    if (properties) payload.properties = properties;
```

Comment:
should we also check that it's an object with key count > 0?

Reply:
the API will check if it is empty before processing